src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  &__item {
+    list-style: none;
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $link-color;
+    font-weight: 500;
+    font-size: 12px;
+    text-transform: uppercase;
+    position: relative;
+    height: 60px;
+    display: flex;
+    align-items: center;
+    transition-duration: .3s;
+  }
+
+  &__link:hover {
+    color: $active-color;
+  }
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: $active-color;
+  border-radius: 8px;
+}
+
+.is-active {
+  color: $active-color;
+}
```

Comment:
The same as above
<img width="1018" alt="Nesting" src="https://github.com/mate-academy/layout_catalog/assets/86789161/eac8b9ce-5ccc-4f37-9916-1e291b71727c">